cmd/api/auth/authn.go

```diff
@@ -9,6 +9,8 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/kubearchive/kubearchive/cmd/api/abort"
```

Comment:
How are all these changes in authn.go related to getting the kind from the resource name?

Reply:
I created a new go module for the middleware layer in charge of doing the request of the api resources called ` discover`. This module needs the `Abort` function so I exported it outside from the `auth` module to its own one called `abort`.
Exporting the `abort` function and moving it to its own module created the need of importing it in the `auth` module to keep things working.

Reply:
Understood.  I missed that link.